server/issue.go

```diff
@@ -415,12 +416,12 @@ func httpAPIGetSearchIssues(ji Instance, w http.ResponseWriter, r *http.Request)
 		MaxResults: 50,
 		Fields:     []string{"key", "summary"},
 	})
+	defer CloseJiraResponse(resp)
```

Comment:
Doesn't the continue only affect the for loop?  If so, I think you can refactor and share some of code in the if/else blocks.

https://github.com/mattermost/mattermost-plugin-jira/blob/d42602692e1602d64a40b75fcc529ffe77cd9c59/server/webhook.go#L120-L130